Base1DBarcode.php

```diff
@@ -318,12 +318,13 @@ public function rotateImage($image, $angel)
      * @param int    $h
      * @param array  $color
      * @param bool   $vertical
+     * @param bool   $transparent
      *
      * @return bool
      *
      * @throws \Exception
      */
-    public function getBarcodePNGPath($code, $type, $name=null, $w=2, $h=30, $color=array(0,0,0), $vertical=false)
+    public function getBarcodePNGPath($code, $type, $name=null, $w=2, $h=30, $color=array(0,0,0), $vertical=false, $transparent = true)
```

Comment:
if the owner of repo cares then for congruency purposes perhaps apply the same changes to "getBarcodePNG" method